pkg/networkservice/connectioncontextkernel/ethernetcontext/macaddress/server_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package macaddress
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ligato/vpp-agent/api/models/linux"
+	"github.com/networkservicemesh/sdk/pkg/networkservice/core/next"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/networkservicemesh/sdk-vppagent/pkg/networkservice/vppagent"
+
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection/mechanisms/kernel"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connectioncontext"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
+)
+
+func TestServerBasic(t *testing.T) {
+	ctx := vppagent.WithConfig(context.Background())
```

Comment:
Please use

https://github.com/networkservicemesh/sdk-vppagent/blob/master/pkg/networkservice/vppagent/server.go#L33

In a chain instead of exposing withConfig.

Reply:
fixed.